### DISCUSSION TITLE
Failed to find a meter

### DISCUSSION THREAD

**Discussion by charliesoap4** [mjmeli/ha-duke-energy-gateway#133] [Q&A] [answered]
I'm getting an error when I load the integration:
" Failed to identify any smart meter with gateway access on your account"

I checked Duke Energy's site about smart meters, and it says if I have an OpenWay meter, then I'm good.  What should I be looking for on my account?

**Comment by mjmeli** [comment, marked as answer]
Hey, I think you have some confusion over the gateway part of this. This integration only works if you have a gateway device, which is separate from your meter and has only been provided as part of an invitation pilot program. Just having an OpenWay meter is not sufficient as Duke does not provide the real time usage from this (the most recent you get is the usage from the previous day, which does not work for real time usage as this integration wants).

**Comment by charliesoap4** [comment]
Understood.  Thank you for the clarification.
